### PR TITLE
README.md: Add short release note about Styx 1.0 API.

### PR DESCRIPTION
**Pull request by mikkokar**
Adding a short notice to inform users about master changing to 1.0, as styx-1.0-dev is being merged to master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ Styx is a programmable reverse proxy for JVM (Java Virtual Machine). It can be u
 as a stand-alone application, or as a platform to build custom networking applications. 
 It is non-blocking, fully asynchronous and event driven therefore very scalable.
 
+### Upcoming Release Note
+
+[Styx 1.0](https://github.com/HotelsDotCom/styx/wiki/Styx-1.0-API) is coming to 
+public release soon. `master` is now following bleeding edge for Styx 1.0.
+Styx 0.7 code is available in `styx-0.7` branch which is in maintenance 
+mode for bug fixes only. Follow the [Styx Wiki](https://github.com/HotelsDotCom/styx/wiki) 
+for latest developments.
```
Comment (kvosper) [inline README.md]: Perhaps the phrasing would work better as:

> [Styx 1.0](https://github.com/HotelsDotCom/styx/wiki/Styx-1.0-API) will be publicly released soon.
> The `master` is now the bleeding edge for Styx 1.0.
> Styx 0.7 code is available in the `styx-0.7` branch which is in maintenance mode (bug fixes only). 
> Follow the [Styx Wiki](https://github.com/HotelsDotCom/styx/wiki) for latest developments.

Comment (mikkokar) [reply README.md]: ack

File: README.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ Styx is a programmable reverse proxy for JVM (Java Virtual Machine). It can be u
 as a stand-alone application, or as a platform to build custom networking applications. 
 It is non-blocking, fully asynchronous and event driven therefore very scalable.
 
+### Upcoming Release Note
+
+Styx 1.0 will be publicly released soon. The `master` is now the bleeding edge for Styx 1.0.
+Styx 0.7 code is available in the `styx-0.7` branch which is in maintenance mode (bug fixes only).
+Follow the [Styx Wiki](https://github.com/HotelsDotCom/styx/wiki) for latest developments.
```
Comment (kvosper) [inline README.md]: "for latest developments." should be "for the latest developments."

Comment (mikkokar) [reply README.md]: fixed

**Pull request merged**